README.md

```diff
@@ -93,6 +106,12 @@ cargo run --bin spacewalk-standalone --release -- --dev
 ./target/release/node-template --dev
 ```
 
+## [`cmd-all` script](./scripts/cmd-all)
```

Comment:
@ebma  added description of the script `cmd-all`.

Reply:
Maybe we should also mention our specific use-case for this, ie. that it's needed to circumvent the issue with the currency crate, etc. Otherwise it's not really clear why we have the need for such a script at all.

Reply:
I updated the [description](https://github.com/pendulum-chain/spacewalk/pull/513/commits/38547a661cc1f3f5a9157dc5a85848f53ff31b37)

Reply:
Nice, I'm happy now 👍